src/main/java/com/github/splendor_mobile_game/websocket/handlers/ServerMessageType.java

```diff
@@ -30,5 +30,7 @@ public enum ServerMessageType {
     END_TURN_RESPONSE,
     GET_TOKENS_RESPONSE,
     NOBLE_RECEIVED_ANNOUNCEMENT,
+    PASS_CONFIRM_RESPONSE,
```

Comment:
So this enum variant `PASS_CONFIRM_RESPONSE` is unnecessary as we don't use it?

Reply:
that's right. Will delete that